package-lock.json

```diff
@@ -1,7601 +0,0 @@
-{
```

Comment:
The package-lock.json was fully deleted, could you revert that and npm install again to get the updated version?